sdk/batch/Microsoft.Azure.Batch/CHANGELOG.md

```diff
@@ -1,22 +1,22 @@
-# Microsoft.Azure.Batch release notes
+# 12.1.0-preview.1
 
-## Changes in 12.0.0
-### Features
+# Changes in 12.0.0
+## Features
 - Added ability to specify a collection of public IPs on `NetworkConfiguration` via the new `PublicIPs` property. This guarantees nodes in the Pool will have an 
   IP from the list user provided IPs.
 - Added ability to mount remote file-systems on each node of a pool via the `MountConfiguration` property on `CloudPool`.
 - Shared Image Gallery images can now be specified on the `VirtualMachineImageId` property of `ImageReference` by referencing the image via its ARM ID.
 - **[Breaking]** When not specified, the default value for `WaitForSuccess` on `StartTask` is now `true` (was `false`).
 - **[Breaking]** When not specified, the default value for `Scope` on `AutoUserSpecification` is now always `Pool` (was `Task` on Windows nodes, `Pool` on Linux nodes).
 
-### Bug fixes
+## Bug fixes
  - Improved various confusing or incomplete documentation.
 
-### REST API version
+## REST API version
 This version of the Batch .NET client library targets version 2019-08-01.10.0 of the Azure Batch REST API.
 
-## Changes in 11.0.0
-### Features
+# Changes in 11.0.0
+## Features
```

Comment:
You didn't consistently change the headings in this file. Either change them all or change the ones above back to match the ones below?

Reply:
I wondered about that. I can go ahead and change everything. My thought was to change what enforced by the tool which is the latest version, since the tools will enforce it from here on out I did not see a need to change existing entries.

Reply:
Yeah we actually direct customers to this file somewhat regularly, it would just be awkward to have it have a weird mismatched format. All the headings should match one another (whatever level of H1/H2 they are)